### ISSUE TITLE
Different modes for PyPen

### ISSUE THREAD

**Issue by EriKWDev** [Canvim/PyPen#17]
We should probably allow for some different modes in PyPen. These modes could include, for example, angle_mode, rectangle_mode and color_mode.

### rectangle_mode
Draw rectangles/squares from the center or from the top-left corner

### angle_mdoe
Use either radians or degrees

### color_mode
if a color is supplied as a tuple/array like ```(20, 20, 20)```, the Color.from_user_input automatically interprets it as rgb(a)-values. A settings.color_mode could change this behaviour and instead interpret them as hsv, hsl or other modes.

This could of course be overwritten if the color is supplied as a string like "rgb(20, 20, 20)" or "hsv(20, 20, 20)".

## Implementation

These could live in settings and be identified using strings or build-in enums. For example: (strings)
```python
settings.rectangle_mode = "center" # or "normal"
```
or with enums
```python
settings.rectangle_mode = RectangleModes.CENTER
```

**Comment by cqann** [comment]
This is a great idea, I believe that the mode as a string is more intuitive and less cumbersome. 
E.g
```
settings.rectangle_mode = "center"
```

**Comment by EriKWDev** [comment]
Yeah I agree. I do however think the enum is less prone to errors like spelling xP and it would be available in intellisense (which strings won't be) , and we wouldn't have to worry about capitalization

**Comment by EriKWDev** [comment]
Perhaps we should have a triangle_mode as well to determine the pivot position of triangles in the future
